java-client/src/main/java/org/opensearch/client/json/JsonpUtils.java

```diff
@@ -59,6 +61,39 @@ public class JsonpUtils {
      *         JSON when advancing to next state.
      * @throws java.util.NoSuchElementException if there are no more parsing states.
      */
+
+    static final JsonProvider DEFAULT_PROVIDER = provider();
+
+    static JsonProvider provider() {
+        return JsonProvider.provider();
+    }
+
+    static final JsonpMapper DEFAULT_JSONP_MAPPER = new JsonpMapperBase() {
+        @Override
+        public JsonProvider jsonProvider() {
+            return DEFAULT_PROVIDER;
+        }
+
+        @Override
+        public <T> void serialize(T value, JsonGenerator generator) {
+            if (value instanceof JsonpSerializable) {
+                ((JsonpSerializable) value).serialize(generator, this);
+                return;
+            }
+
+            throw new JsonException(
+                "Cannot find a serializer for type " + value.getClass().getName() + ". Consider using a full-featured JsonpMapper"
```

Comment:
Does this need a period at the end (check other errors)?

Reply:
I found many error messages not ending with a "." , so I went with that, but I am adding it because I think it's more neat that way.

Reply:
Would be great to standardize in a separate PR, wink wink :)